core/clients/metadataclients/metadata-clients.go

```diff
@@ -193,7 +194,7 @@ func (a *AddressableClient) AddressableForName(name string) (models.Addressable,
 	}
 	defer resp.Body.Close()
 	if err != nil {
-		fmt.Println("AddressableForName makeRequest failed: %s", err)
+		fmt.Printf("AddressableForName makeRequest failed: %v\n", err)
```

Comment:
Do we need hese printf lines? I would like to remove all of these in the production and use only logs.

Reply:
I agree with you. But in this case I just wanted to remove the vet warnings.

We will need to cleanup all the codebase logs.

Reply:
Agreed on this.